### PR TITLE
Update english part for hackathon.

### PR DESCRIPTION
**Pull request by jeremy0519**
Issue #315

### CODE REVIEW

**Comment by jeremy0519** [comment]
Don't merge it until it is completed (remaining sections are completed).

**Comment by jeremy0519** [comment]
These are the sections needed to be reviewed:
- [x] participant
- [x] subject
- [x] activity
- [x] holder
- [x] application

**Comment by zhaofeng-shu33** [comment]
You have completed the translation of `participates` before. So where is it?

**Comment by jeremy0519** [comment]
I fixed it just now.

**Review by zhaofeng-shu33 — COMMENTED** [review]
ok

**Comment by jeremy0519** [comment]
Can you translate the remaining 2 sections?

File: i18n/en-us.js [hunk 1]
```diff
@@ -278,26 +278,49 @@ export default {
       ],
     },
     participant: {
-      name: '',
```
Comment (jeremy0519) [inline i18n/en-us.js:281]: Do you think we need to capitalize the first letter?

Comment (zhaofeng-shu33) [reply i18n/en-us.js:281]: Yes

**Comment by zhaofeng-shu33** [comment]
> Can you translate the remaining 2 sections?

ok

**Comment by zhaofeng-shu33** [comment]
I have finished the remaining trans for Hackathon. It seems that current pc css stylesheet does not work well with English page. @lovepoem @jeremy0519

**Comment by jeremy0519** [comment]
I have no idea about the technical things.

**Review by zhaofeng-shu33 — APPROVED** [review]

**Pull request merged**